dashboard/v1.1/src/services/input.ts

```diff
@@ -0,0 +1,42 @@
+export const defaultHTTPMethodsList = [
+  {
+    name: 'HTTP',
+    value: 'HTTP'
+  },
+  {
+    name: 'HTTPS',
+    value: 'HTTPS'
+  },
+  {
+    name: 'Manual',
+    value: 'Manual'
+  },
+];
+
+export const defaultRequestMethodsList = [
+  {
+    name: 'GET',
+    value: 'GET'
+  },
+  {
+    name: 'POST',
+    value: 'POST'
+  },
+  {
+    name: 'PUT',
+    value: 'PUT'
+  },
+  {
+    name: 'DELETE',
+    value: 'DELETE'
+  },
+  {
+    name: 'PATCH',
+    value: 'PATCH'
+  }
+];
+
+export const defaultLabel = {
+  name: 'no label',
+  color: '#e5e5e5'
+}
```

Comment:
This too as `enum`.

Reply:
This is a exported default object to a state